src/main/java/com/github/susom/dbgoodies/etl/BigQueryWriter.java

```diff
@@ -384,10 +389,12 @@ public T process(Rows rs) throws Exception {
                 totalCnt.addAndGet(batchInsertData.size());
                 long timeTook = (Instant.now().getEpochSecond()-lastBatchStartTs);
                 long totalTimeTook = Instant.now().getEpochSecond() - startTs;
-                if(timeTook!=0)
-                  log.info("db "+tableName+" total:"+totalCnt.get()+" time "+timeTook+" s speed:"+ totalCnt.get()/totalTimeTook +" r/s");
-                else
+                if(timeTook!=0) {
+                  log.info("db " + tableName + " total:" + totalCnt.get() + " time " + timeTook
+                      + " s speed:" + totalCnt.get() / totalTimeTook + " r/s");
+                }else {
                   System.out.print("*");
+                }
```

Comment:
I understand you didn't change it in this PR, but writing console output here doesn't seem right...

Reply:
I can comment this out. It is to show if the uploader is actively running rather than silently hanging, and using standard logging is not feasible for this. 

It was necessary while developing the uploader earlier on, but it is fine to hide this since this tool is very stable now.